src/test/java/com/google/api/codegen/discogapic/testdata/java/java_simplecompute.v1.json.baseline

```diff
@@ -8928,9 +8928,9 @@ public class AddressClient implements BackgroundResource {
    * Sample code:
    * <pre><code>
    * try (AddressClient addressClient = AddressClient.create()) {
-   *   ProjectName project = ProjectName.of("[PROJECT]");
+   *   String formattedProject = ProjectName.format("[PROJECT]");
```

Comment:
Are we expecting these changes in the compute baselines? Will this affect the actual compute client?

Reply:
Yes, these changes will be in the actual compute client too.

No, these don't affect the client because (1) they are only comment changes and (2) they have equivalent behaviors.